src/scenes/Event/scenes/oracle.js

```diff
@@ -182,6 +165,39 @@ class OraclePage extends React.Component {
     ];
   }
 
+  isActionButtonDisabled() {
+    const {
+      currentOptionIdx,
+      voteAmount,
+      config,
+      transactions,
+      address,
+    } = this.state;
+
+    // Not within correct times or not the result setter
+    if (config.predictionAction.btnDisabled) {
+      return true;
+    }
+
+    // Already have a pending tx for this Oracle
+    const pendingTxs = _.filter(transactions, { oracleAddress: address, status: TransactionStatus.Pending });
+    if (pendingTxs > 0) {
+      return true;
+    }
+
+    // Did not select an option
+    if (currentOptionIdx === -1) {
+      return true;
+    }
+
+    // Did not enter an amount
+    if ((voteAmount === 0 || Number.isNaN(voteAmount)) && config.predictionAction.showAmountInput) {
```

Comment:
This will cause a but in finalize and set result stage

Reply:
this `if` will return false during those two stages because of `config.predictionAction.showAmountInput`